src/server.tsx

```diff
@@ -376,6 +380,34 @@ app.post('/epic/compare-variant-decision', async (req: express.Request, res: exp
     res.send('thanks');
 });
 
+app.get(
+    '/amp/experiments_data',
+    cors({
+        origin: [
+            'https://amp-theguardian-com.cdn.ampproject.org',
+            'https://amp.theguardian.com',
+            'http://localhost:3030',
+            'https://amp.code.dev-theguardian.com',
+        ],
+        credentials: true,
+        allowedHeaders: ['x-gu-geoip-country-code'],
+    }),
+    async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+        try {
+            const response = await getAmpExperimentData();
+
+            // The cache key in fastly is the X-GU-GeoIP-Country-Code header
+            res.setHeader('Surrogate-Control', 'max-age=300');
+            res.setHeader('Cache-Control', 'max-age=60');
+            res.setHeader('Content-Type', 'application/json');
+
+            res.send(JSON.stringify(response));
```

Comment:
I think we can swap this out for a call to `res.json(response)`

Reply:
✅